tests/Integration/Behaviour/Features/Scenario/Product/update_stock_advanced.feature

```diff
@@ -240,14 +240,14 @@ Feature: Update product stock from Back Office (BO)
       | quantity | 51 |
     And product "product1" last stock movement has following details:
       | physical_quantity | 51 |
```

Comment:
i thought it would become something like this:
```
      | physical_quantity | +51 |
      | physical_quantity | -51 |
```

Reply:
ooohhh